pages/faq.tsx

```diff
@@ -13,59 +13,45 @@ export default function FAQ(): ReactElement {
         <h2>Frequently Asked Questions</h2>
 
-        {/* What is loot ? */}
+        {/* What is PROJECT ? */}
         <div className={styles.faq__item}>
-          <h3>What is Loot?</h3>
+          <h3>What are Adventure Cards?</h3>
           <p>
-            Loot is a collection of 8,000 unique bags of adventurer gear,
+            Adventure Cards is a collection of 8,000 unique card decks,
             originally released by{" "}
             <a
-              href="https://twitter.com/dhof/status/1431316631934967815"
+              href="https://twitter.com/pm/status/1440358659536556040?s=20"
               target="_blank"
               rel="noopener noreferrer"
             >
-              Dom Hofmann
+              Paul McKellar
             </a>
-            . At release, anyone could claim loot bags for just gas, and all
-            bags were claimed in under 4 hours. Each loot bag contains 8 items:
-            a piece for an adventurer&apos;s chest, foot, hand, head, neck, ring,
-            waist, and weapon.
+            . At release, anyone could claim up to 2 decks for just gas, and the
+            first decks were claimed in under 14 minutes. Each deck includes 45 
+            procedurally generated cards, enough to play a full game:
+            common cards (for playability) and rares to keep it interesting.
           </p>
           <p>
-            Loot is an unaudited project. Bags #1 to #7777 were claimable by
-            anyone and #7778 to #8000 are currently reserved for the contract
-            deployer.
+            Adventure Cards is an unaudited project. Bags #0 to #4887 were claimable by
+            anyone and #7778 to #8000 are currently reserved for the owner.
           </p>
         </div>
 
-        {/* Why is loot special? */}
+        {/* Why is PROJECT special? */}
         <div className={styles.faq__item}>
-          <h3>Why is loot special?</h3>
+          <h3>Why are Adventure Cards special?</h3>
           <p>
-            Loot is unique—the first project of its kind. With no company, art,
-            team, or attributes, Loot makes it impossible to gate-keep any
-            creative decisions (h/t{" "}
-            <a
-              href="https://twitter.com/john_c_palmer/status/1432606797186179072?s=20"
-              target="_blank"
-              rel="noopener noreferrer"
-            >
-              @john_c_palmer
-            </a>
-            ).
-          </p>
-          <p>
-            Loot is the unfiltered, uncensorable building block for stories,
-            experiences, games, and more, in the hands of the community, at no
-            cost. Loot pursues complete decentralization from day one.
+            Adventure Cards is the first collectable card game stored entirely on chain.
+            Each NFT includes 45 procedurally generated cards, enough to play a full game.
+            Inspired by LOOT and MtG, Adventure cards are crypto primitives.
```

Comment:
maybe spell out Magic: The Gathering for the kids in the back?

Reply:
same copytext as the OG https://0xadventures.com/ fwiw

Reply:
it's wonderful reading this